sonarqube.bash

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
What would you think about hiding this file in a `scripts/` folder or something? Like I can understand having `pr_check.sh` and `build_deploy.sh` in the root since jenkins needs them - but this one doesn't need to be in the root necessarily?

Reply:
Yeah, I agree. Moving it now!